it/mail/chrome/messenger/filter.properties

```diff
@@ -1,71 +1,71 @@
+pickTemplateToReplyWith=Scegli un modello con cui rispondere.
+mustEnterName=Devi assegnare un nome a questo filtro.
+mustSelectAction=Devi selezionare almeno un'azione filtro.
+cannotHaveDuplicateFilterTitle=Nome filtro duplicato
+cannotHaveDuplicateFilterMessage=Il nome filtro inserito seiste già. Inserisci un nome filtro diverso.
+deleteFilterConfirmation=Sei sicuro di voler eliminare il filtro o i filtri selezionati?
+untitledFilterName=Filtro senza titolo
+filterListBackUpMsg=I tuoi filtri non funzionano perché il file msgFilterRules.dat, che contiene i tuoi filtri, non è leggibile. Sarà creato un nuovo file msgFilterRules.dat; e sarà creata una copia del file originale, nominata rulesbackup.dat, nella stessa cartella.
+customHeaderOverflow=Hai superato il limite di 50 intestazioni personalizzate. Rimuovi una o più intestazioni personalizzate e riprova.
+filterCustomHeaderOverflow=I tuoi filtri hanno superato il limite di 50 intestazioni personalizzate. Modifica il file msgFilterRules.dat, che contiene i tuoi filtri, per utilizzare meno intestazioni personalizzate.
+invalidCustomHeader=Uno dei tuoi filtri utilizza un'intestazione personalizzata che contiene un carattere non valido, come ':', un carattere non stampabile, un carattere non ASCII o un carattere ASCII a 8 bit. Modifica il file msgFilterRules.dat, che contiene i tuoi filtri, per rimuovere i caratteri non validi dalle intestazioni personalizzate.
+continueFilterExecution=L'applicazione del filtro %S non è riuscita. Vuoi continuare ad applicare i filtri?
+promptTitle=Esecuzione filtri
+promptMsg=Stai attualmente filtrando i messaggi.\nVuoi continuare ad applicare i filtri?
 stopButtonLabel=Interrompi
 continueButtonLabel=Continua
-cannotEnableFilter=Questo filtro è stato probabilmente creato da una versione successiva di Mozilla/Netscape. Non è possibile attivare questo filtro in quanto il formato non è supportato.
+cannotEnableFilter=Il filtro è stato probabilmente creato da una versione successiva di mozilla/netscape. Non puoi attivare questo filtro in quanto il formato non è supportato.
 
 # for junk mail logging / mail filter logging
 # LOCALIZATION NOTE(junkLogDetectStr)
 # %1$S=author, %2$S=subject, %3$S=date
 junkLogDetectStr=Rilevata posta indesiderata da %1$S - %2$S a %3$S
 # LOCALIZATION NOTE(logMoveStr)
 # %1$S=message id, %2$S=folder URI
-logMoveStr=spostato l'id del messaggio = %1$S a %2$S
+logMoveStr=spostato id del messaggio = %1$S a %2$S
 # LOCALIZATION NOTE(logCopyStr)
 # %1$S=message id, %2$S=folder URI
-logCopyStr=copiato l'id del messaggio = %1$S a %2$S
+logCopyStr=copiato id del messaggio = %1$S a %2$S
 # LOCALIZATION NOTE(filterLogDetectStr)
 # %1$S=filter name %2$S=author, %3$S=subject, %4$S=date
 filterLogDetectStr=Applicato filtro "%1$S" al messaggio da %2$S - %3$S a %4$S
-filterAction2=cambiata la priorità
+filterAction2=priorità modificata
 filterAction3=eliminato
-filterAction4=marcato come letto
+filterAction4=contrasseganto come letto
```

Comment:
CORRECTIONS NEEDED. 

There is a typo. Please replace "contrasseganto" with "contrassegnato".